website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,102 @@
+
+### Accordion vs Reveal
+
+While similar in functionality and interaction, the Accordion and Reveal are meant to be used in different scenarios and use cases.
+
+#### When to use the Accordion
+
+- You have multiple sections of content that can be expanded or collapsed.
+
+- Each section of content has its own toggle content, giving users the ability to access specific sections.
+
+- Users need to navigate through different sections of content and selectively expand or collapse sections
+
+- There is a sequential or hierarchical relationship between the different sections.
+
+#### When to use the Reveal
+
+- You have a single piece of content that can be expanded to collapsed to reveal additional details or information.
+
+- The content within the toggle is plain text.
+
+- The content within the reveal is relatively short and simple.
+
+- There is no need for multiple independent sections or hierarchical relationship between different sections.
+
+### Usage examples
+
+!!! Do
+
+Use the Accordion when you need to display content in different sections that have a sequential or hierarchical relationship between them.
+
+![Accordion with basic content](/assets/components/accordion/usage-do.png)
+!!!
+
+!!! Dont
+
+Don’t use the Accordion when you need to display a single piece of content that can expand and collapse. Use the [Reveal](/components/reveal) instead.
+
+![Accordion with basic content](/assets/components/accordion/usage-do-not.png)
+!!!
+
+!!! Do
+
+Use the Accordion when the toggle or the content is relatively complex.
+
+![Accordion with complex content](/assets/components/accordion/usage-do-complex.png)
+!!!
+
+## Toggle content
+
+The toggle accepts many different types of content, from text-based content to nested HDS components or custom content.
+
+![Example of toggle content](/assets/components/accordion/toggle-content-text.png =690x*)
+
+### Generic content
+
+![Example of toggle content](/assets/components/accordion/toggle-content-custom.png =690x*)
+
+### Interactive content
+
+Use `isInteractive` when nesting interactive content on the toggle.
+
+![Example of toggle content](/assets/components/accordion/toggle-content-interactive.png =690x*)
+
+## Content type
+
+!!! Info
+
+The content type property is only relevant within Figma and doesn’t exist as a property in the code.
+!!!
+
+The `content` supports any custom content, local components, or Helios components via an `instance swap` property (customInstance) in Figma. In code, `yield` is supported.
+
+### Default
+
+![Example of content in accordion item](/assets/components/accordion/content-text.png =690x*)
+
+### Custom
+
+![Example of content in accordion item](/assets/components/accordion/content-custom.png =690x*)
+
+## isOpen
+
+In code, all accordion items are collapsed by default. Pass `isOpen` to display a single item open on page load.
+
+![Example of content in accordion item](/assets/components/accordion/accordion-item-open.png =800x*)
```

Comment:
This isn't necessarily usage guidance but more technical guidance for setting an open section in the code, which is already covered in the Code tab. We should probably remove this so it doesn't need to be maintained in two sections.